lib/y-richtext.coffee

```diff
@@ -172,12 +172,20 @@ class YRichText extends BaseClass
 
       if delta.insert?
         insertHelper thisObj, position, delta.insert
-        from = thisObj._model.getContent("characters").ref(position)
-        to = thisObj._model.getContent("characters").ref(position+delta.insert.length)
-        thisObj._model.getContent("selections").select from, to, delta_selections
-        thisObj._model.getContent("selections").unselect from, to, delta_unselections
-
-        return position + delta.insert.length
+        length =
```

Comment:
Furthermore, if `delta.insert` is a number/object/not a string, it will get transformed to a string anyway by the current implementation